src/lib.rs

```diff
@@ -181,3 +183,63 @@ impl<T> From<T> for SyncWrapper<T> {
         Self::new(value)
     }
 }
+
+
+pub struct SyncFuture<F> {
+    inner: SyncWrapper<F>
+}
+impl <F: Future> SyncFuture<F> {
+    /// Create a `Future` which is `Sync`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use sync_wrapper::{SyncWrapper, SyncFuture};
+    ///
+    /// let fut = async { 1 };
+    /// let fut = SyncFuture::new(fut);
+    /// ```
```

Comment:
I usually put these docs on the struct itself, which has the added benefit of showing the first line in the summary of the docs.